common/changes/@microsoft/tsdoc/patch-1_2023-05-03-18-55.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/tsdoc",
+      "comment": "Improve the docs to clarify that overload indexes count from 1 not 0",
```

Comment:
@rugvip I pushed an update to your branch that improves the docs. Let me know what you think.